app/lib/package.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 require 'zip'
-require 'page_client'
-require 'xml_util'
 
 class Package
+  XPATH_RESOURCES = %w[//@background-image
+                       //manifest:manifest/@banner
```

Comment:
this one should be `//manifest:category/@banner`
https://github.com/CruGlobal/mobile-content-api/blob/master/public/xmlns/manifest.xsd#L140